renku/cli/image.py

```diff
@@ -17,17 +17,43 @@
 # limitations under the License.
 """Manipulate images related to the Renku project.
 
+Configure registry
```

Comment:
"Configure the image registry"

The target demographic of Renku may not know that registry and repository are two different things or that the registry even exists.